trainers/sekiro.toml

```diff
@@ -0,0 +1,29 @@
+name = "Sekiro Trainer"
+version = '0.0.1'
+process = "sekiro.exe"
+enable = true
```

Comment:
Due to how the daemon works it's important everything (including features) are disabled by default since all the trainers are automatically installed in the arch AUR package, though this might change in the future.